server/config.ts

```diff
@@ -107,5 +107,6 @@ export default {
   environmentName: get('ENVIRONMENT_NAME', ''),
   flags: {
     maintenanceMode: get('IN_MAINTENANCE_MODE', false),
+    enablePrisonDashboard: get('ENABLE_PRISON_DASHBOARD', false) === 'true',
```

Comment:
am i reading this right that the value in the YAML is a string but the default value we're handing in to `get()` is a boolean? I guess it should work either way but instinctively I don't like it. Is there any way around it?

Reply:
It's a good point, we should use string comparison instead to match [other usages](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-bail-ui/blob/main/server/config.ts#L103).